lib/gds_api/publishing_api.rb

```diff
@@ -34,6 +36,31 @@ def get_content(content_id, params = {})
     get_json(content_url(content_id, params))
   end
 
+  # Return a live content item
+  #
+  # Raises exception if the item doesn't exist.
+  # Raises an exception if the version number is passed in the params
+  #
+  # @param content_id [UUID]
+  # @param locale [String]
+  # @option locale [String] locale The language, defaults to 'en' in publishing-api.
```

Comment:
How come we have a @param and @option?

Reply:
Oops! I'll remove `@param` as locale is optional.